readme.md

```diff
@@ -45,4 +45,39 @@ composer install
 ## General
 
 - PHP version 8.1
-- Use strict types `declare(strict_types=1);`
\ No newline at end of file
+- Use strict types `declare(strict_types=1);`
+
+***
+
+## Example Ecommerce store
```

Comment:
We need to include the built binaries and instructions to run these binaries.
We should not expect the developer to know about Go or even have it installed.

You can keep the below instructions, perhaps lets move them to the example store directory...

Reply:
> We need to include the built binaries

Keeping the binaries in the repo is not a good idea. If you have to make build binaries available, you can 
- link to a public S3 bucket with the binaries
- upload the binaries when [tagging a release](https://github.com/stock2shop/connector/releases)